website/docs/advanced/search_path.md

```diff
@@ -3,15 +3,94 @@ id: search_path
 title: Config Search Path
 ---
 
-The Config Search Path is a list of paths that are searched in order to find configs. It is similar to
-the Python PYTHONPATH.
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
 
+The Config Search Path is a list of paths that Hydra searches in order to find configs. It is similar to
+the Python `PYTHONPATH`. 
  - When a config is requested, The first matching config in the search path is used.
- - Each search path element has a schema prefix such as file:// or pkg:// that is corresponding to a ConfigSourcePlugin.
- - `SearchPathPlugin` can manipulate the search path.
+ - Each search path element has a schema prefix such as `file://` or `pkg://` that is corresponding to a `ConfigSourcePlugin`.
 
 You can inspect the search path and the configurations loaded by Hydra by turning on verbose logging for the `hydra` logger:
 
-```text
+```bash
 $ python my_app.py hydra.verbose=hydra
-```
\ No newline at end of file
+```
+
+There's a few options to modify the config search path so Hydra could discover 
+the configurations. You can pick and choose some or all of the following methods below
+to provide configuration to your apps.
+
+### Set `config_name`/`config_path` in`hydra.main()`
+This is the most common use case: all config files are grouped under one directory and
+has a relative path to the application. You can add them to the search path by setting `config_path` 
+or `config_name`.
+
```

Comment:
1. config_name should **absolutely** not be used to add a directory to the search path.
2. style wise, the title looks huge (look at the resulting page).

Reply:
Fixed the text & downsized the title